modules/local/samplesheet_check.nf

```diff
@@ -2,10 +2,10 @@ process SAMPLESHEET_CHECK {
     tag "$samplesheet"
     label 'process_single'
 
-    conda "conda-forge::python=3.8.3"
+    conda "conda-forge::python=3.9.16"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/python:3.8.3' :
-        'biocontainers/python:3.8.3' }"
+        'https://depot.galaxyproject.org/singularity/python:3.9.16' :
+        'biocontainers/python:3.9.16' }"
```

Comment:
Apparently version 3.9.16 of python is not available in the bioconductors repository: https://quay.io/repository/biocontainers/python?tab=tags
They have 3.9 or 3.9--1, not sure what's the latter?

Reply:
Oh, of course, forgot to check that... Downgraded to `3.9` now, which works on both Conda and containers.